assets/scripts/script.js

```diff
@@ -84,5 +117,51 @@ const implex = (function () {
     }
   }
 
+  function onSubmit(e) {
+    e.preventDefault();
+
+    const xhr = new XMLHttpRequest();
+    const nameInputField = document.getElementById('name-input');
+    const emailInputField = document.getElementById('email-input');
+    const commentInputField = document.getElementById('comment-input');
+    let nameInputValue = nameInputField.value;
+    let emailInputValue = emailInputField.value;
+    let commentInputValue = commentInputField.value;
+
+    const errorBlock = document.getElementsByClassName('error-block')[0];
+    const successBlock = document.getElementById('success-block');
+
+    const FD = new FormData(form);
+
+    if (nameInputValue === '' || emailInputValue === '' || commentInputValue === '') {
+      errorBlock.style.display = 'flex';
+      errorBlock.textContent = 'Please enter valid value';
```

Comment:
What is the sense of adding text here if it is the only place where content changes?
Why not hardcode it?